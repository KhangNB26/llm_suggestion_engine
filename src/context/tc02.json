{
  "workItems": [
    {
      "work_id": "11111111-aaaa-bbbb-cccc-000000000002",
      "task_id": "11111111-aaaa-bbbb-cccc-000000000010",
      "title": "Checklist: Lam baitap Vat ly",
      "effective_deadline": "2025-09-27T09:00:00Z",
      "effective_priority": 2,
      "status": 1,
      "is_atomic": false,
      "task_deadline": "2025-09-28T09:00:00Z",
      "task_priority": 2,
      "task_status": 1
    }
  ],
  "scheduleEntries": [],
  "freeSlots": [
    {
      "startUtc": "2025-09-25T09:00:00Z",
      "endUtc": "2025-09-25T11:00:00Z",
      "startLocal": "2025-09-25T16:00:00+07:00",
      "endLocal": "2025-09-25T18:00:00+07:00",
      "durationMinutes": 120,
      "timeZone": "Asia/Ho_Chi_Minh"
    }
  ],
  "studyProfile": {
    "user_id": "11111111-1111-1111-1111-111111111111",
    "chronotype": "early_bird",
    "focusStyle": "deep_work",
    "workStyle": "solo"
  }
}
