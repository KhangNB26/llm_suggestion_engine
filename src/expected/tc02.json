{
  "items": [
    {
      "item_type": 1,
      "title": "Lam baitap Vat ly - Checklist moi",
      "parentTaskId": null,
      "estimated_minutes": 60,
      "deadline": "2025-09-25T15:00:00Z",
      "confidence": 0.8,
      "reason": "Tach cong viec thanh checklist rieng, phu hop slot trong 2h."
    }
  ]
}
